lang/lang_en_fr.txt

```diff
@@ -234,10 +262,26 @@
 "Error - static memory has been overwritten"
 "Erreur - la memoire statique a ete ecrasee"
 
+#MSG_CUT_FILAMENT c=16
+"Cut filament"
+"Coupe filament"
+
+#MSG_CUTTER c=9
+"Cutter"
+"Coupeur"
+
+#c=18
+"Cutting filament"
+"Coupe filament"
```

Comment:
"Coupez filament"

Reply:
`Coupez filament` is a command to cut the filament. In this case, the printer is doing the cutting, so a better translation would be `Je coupe filament`.

Reply:
"Coupe du filament" ?

Reply:
I believe that this is an informative message and is displayed when the MMU is cutting filament. I interpret `coupe du filament` as "cut some filament" or "cut of filament". Really all we're doing with this message is telling the user that the printer is doing work. I cannot think of an alternative way to tell the user that the printer is working while staying within the limits.

Reply:
I believe the best option here is `Je coupe filament`.

Reply:
> Je coupe filament

Use this